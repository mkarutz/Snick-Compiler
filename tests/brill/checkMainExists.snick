proc foo ()
    int x;
    write x;
end
