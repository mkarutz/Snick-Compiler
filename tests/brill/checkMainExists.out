Failure: No main procedure definition.
Compilation Failed
